ui/src/api/aws.js

```diff
@@ -53,6 +53,15 @@ export function fetchPlates(experimentId, status) {
 }
 
 export function saveExperimentPlates(experimentName, status, plateMaps) {
+  return new Promise((resolve, reject) => {
+    let plateMapsToSave = lzutf8.compress(JSON.stringify(plateMaps), {
+      outputEncoding: 'Base64',
+    });
+      saveToDB(experimentName, status, 0, plateMapsToSave, reject, resolve);
+  });
+}
+
+export function publishExperimentPlates(experimentName, status, plateMaps) {
   return new Promise((resolve, reject) => {
     let plateMapsToSave = lzutf8.compress(JSON.stringify(plateMaps), {
       outputEncoding: 'Base64',
```

Comment:
Why are we even still checking the status? They should be two different functions?

Reply:
Checking the status in case something went wrong and the status wasnt changed. If the status somehow remained draft, then the version in dynamo would be saved as draft with a versioned timestamp.

Reply:
But you are logging an error... If  you make it explicit what publish does, I don't think you need to include the check and also log the error. We effectively are removing the need for having a status.

Reply:
Retained the statuses of the plates but no longer use the current status to determine whether to save draft of publish completed.  Now that behavior is determined by the "mark as complete" 
button press.